acknowledge_license_cert.html

```diff
@@ -267,10 +267,10 @@
 <tr>
 <td>
 Open Med Practice</td>
-<td><a href="http://ww2.openmedpractice.com/">ww2.openmedpractice.com</a><br>
-USA<br>
+<td><a href="http://www.openmedpractice.com/">www.openmedpractice.com</a><br>
+505 Independance Parkway, Chesapeake, VA 23320<br>
 Phone: +1 757 328 2736<br> 
-Sherwin Gaddis: sgaddis@jse.net</td>
+Sherwin Gaddis: sherwin@openmedpractice.com</td>
 </tr>
```

Comment:
Ok, Great! I just updated the Wiki page.

Sherwin
757-328-2736
Open Med Practice

       www.openmedpractice.com <http://ww2.openmedpractice.com/>

OpenEMR Simplicity at its best in cloud based EMR's


“For those who believe, no proof is necessary. For those who don't believe,
no proof is possible.” ― Stuart Chase
<http://www.goodreads.com/author/show/1051789.Stuart_Chase>

On Mon, May 23, 2016 at 6:10 AM, Brady Miller <notifications@github.com>
wrote:

> In acknowledge_license_cert.html
> <https://github.com/openemr/openemr/pull/127#discussion_r64198121>:
>
> >  Phone: +1 757 328 2736<br>
> > -Sherwin Gaddis: sgaddis@jse.net</td>
> > +Sherwin Gaddis: sherwin@openmedpractice.com</td>
> >  </tr>
>
> Note I bring in any changes or new entries on the above wiki page into the
> acknowledge_license_cert.html script (this ensure they are the same).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openemr/openemr/pull/127/files/c2fac0dc00d9ff4b62e77c4ca15273888d13bfe6#r64198121>
>